docs/ref/models/fields.txt

```diff
@@ -12,11 +12,14 @@ This document contains all the API references of :class:`Field` including the
 
 .. seealso::
 
-    If the built-in fields don't do the trick, you can try `django-localflavor
-    <https://django-localflavor.readthedocs.org/>`_, which contains assorted
-    pieces of code that are useful for particular countries or cultures. Also,
-    you can easily :doc:`write your own custom model fields
-    </howto/custom-model-fields>`.
+   If the built-in fields don't do the trick, you can refer to the
+   package ``django-localflavor``, which contains assorted pieces of code
```

Comment:
I prefer the inline link as originally done rather than mixing raw URLs in the text.